brave-lists/clean-urls.json

```diff
@@ -213,5 +213,16 @@
             "brave_testing1",
             "brave_testing2"
         ]
-    }
+    },
+    {
+        "include": [
+            "*://medium.com/*"
+        ],
+        "exclude": [
+
+        ],
+        "params": [
+            "source"
+        ]
+    },
```

Comment:
This will make the JSON file invalid because trailing commas are not allowed, unfortunately. I suggest simply moving this new block to be just below the YouTube block (around line 147). That means there's no need to add the annoying comma at the end, and we keep the test entry last.